ci/travis/astyle.sh

```diff
@@ -43,7 +43,7 @@ fi
 for file in $FILES; do
 	if is_source_file $file
 	then 
-		./build/astyle/build/gcc/bin/astyle --options=./ci/scripts/astyle_config $file
+		./build/astyle/build/gcc/bin/astyle --options=$ASTYLE_CONFIG_PATH $file
```

Comment:
why not use `"$(get_script_path astyle_config)` here ?

Reply:
Because currently 'lib.sh' is used only by other repos that require same scripts. 

Currently seems pretty redundant to use it in this repo, but I can add it for the sake of similarity.